tgui/packages/tgui/interfaces/IntegratedCircuit/ComponentMenu.js

```diff
@@ -133,6 +132,19 @@ export class ComponentMenu extends Component {
           </Stack.Item>
           <Stack.Item>
             <Stack vertical fill>
+              {trueLength === 0 && (
+                <Stack.Item mt={1} fontSize={1}>
+                  <NoticeBox info>
+                    You can hit this integrated circuit onto a component printer
+                    to link it so that you&apos;re able to remotely create and
+                    add components to this circuit.
+                    <br />
+                    <br />
+                    Close and re-open the integrated circuit interface once
+                    linked to fully complete the linking.
```

Comment:
Why is this needed? Will refreshing static data fix it so this won't be necessary?

Reply:
yeah, if static data is refreshed